Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml

```diff
@@ -107,6 +107,33 @@
     proto: RedLaser
     fireCost: 62.5
 
+- type: entity
+  name: syndicate laser gun
+  parent: BaseWeaponBattery
+  id: WeaponSyndieLaserCarbine
```

Comment:
Just call it a laser gun to match the name. IDs and names that don't match are annoying

Reply:
tbf the original is also called a "carbine" in its item ID but i'll change it